Assets/Scripts/Server/GameMaster/GameMasterBehavior.cs

```diff
@@ -87,6 +95,12 @@ private void SpawnNewTetrominosIfNeeded()
     foreach (var player in _players)
     {
       var tetromino = _tetrominoFactory.SpawnTetromino(distanceBetweenPlayerSpawns * i++);
+      if(tetromino == null)
+      {
+        //GAME OVER

```

Comment:
Please leave a comment 'splain' this.

"If a tetrimino couldn't be spawned, it is assumed to be because there is a block blocking the spawn point. If that's the case, it means that the player LOSES"

or something.